.ci/update-tekton-bundle.sh

```diff
@@ -0,0 +1,87 @@
+APPSTUDIO_TASKS_REPO=quay.io/repository/redhat-appstudio-qe/test-images
+PIPELINE_BUNDLE_IMG=quay.io/repository/redhat-appstudio-qe/test-images:pipeline-bundle-$JVM_BUILD_SERVICE_PR_SHA
+BUILD_DEFINITIONS_DIR="$WORKSPACE"/tmp/build-definitions
+
+function cloneBuildDefinitions() {
+    git clone https://$GITHUB_TOKEN@github.com/redhat-appstudio/build-definitions.git "$BUILD_DEFINITIONS_DIR"
+}
+
+function updateAnalyzerImage() {
+    yq e -i "(.[] \
+    | select(.path == \"/spec/steps/-\").value \
+    | select(.name == \"analyse-dependencies-java-sbom\").image) \
+    |= \"$JVM_BUILD_SERVICE_ANALYZER_IMAGE\"" \
+    $BUILD_DEFINITIONS_DIR/tasks/patches/add-sbom-and-push.yaml
+}
+
+function createDockerConfig() {
+    mkdir ~/.docker
+    echo $QUAY_TOKEN | base64 -d > ~/.docker/config.json
+}
+
+function createAndPushTaskBundle() {
```

Comment:
I would go with way of getting current bundles, storing into file, modify and upload again:
```
tkn bundle list -o yaml quay.io/redhat-appstudio/build-templates-bundle:0e0b28716550e1adc50443084adecec67f1748ef > pipelines.yaml
for bundle in $(grep bundle pipelines.yaml  | sort -u | awk '{print $2}'); do i=`echo $bundle | sed 's/.*-//'`; tkn bundle list -o yaml $bundle > tasks-$i.yaml; done
```
Find the task, modify it, upload new bundle (could be enough to upload new bundle only with that one task). Modify pipelines.yaml -> change task bundle link for your modified task.

Reply:
Any suggestions on how to get the latest tag?

Reply:
from infra-deployments repository which is available (since it's running preview script)